src/index.tsx

```diff
@@ -54,8 +64,29 @@ initialize({
   ],
   componentWrappers: {
     SiteHeader: () => Header,
+    Page: (DefaultComponent: React.ComponentType<Props>) => {
+      return function ClinEpiPage(props: Props) {
+        useEffect(() => {
+          if (process.env.REACT_APP_ENABLE_DATA_RESTRICTIONS === 'true') {
```

Comment:
Would a fail-safe approach be better?  The restrictions are on unless devs explicitly turn them off?

I know this might cause short-term pain when we first pull the changes into our dev instance, but it's better than forgetting to turn it on in a production instance, maybe?

Reply:
In fact, reverting my most recent commit will change the behavior to the fail-safe approach you described! I'll go with my first instincts, because the short-term pain isn't all that painful.

I should clarify that: 

1. This component is the `Page` that's rendered by the dev server - the production version of the `Page` currently lives [here](https://github.com/VEuPathDB/ClinEpiWebsite/blob/master/Site/webapp/js/client/component-wrappers/Page.tsx).
2. In this repo, assets which lie outside `src/lib`, such as this file, aren't published to npm.